build_ext/build_ext/lint.py

```diff
@@ -15,8 +15,8 @@
 from distutils.spawn import spawn
 
 from build_ext.utils import Utils, BaseCommand
-import os
 import subprocess
+import os
```

Comment:
if you move this before 'subprocess', we can avoid an extra change

Reply:
done